students/matt_casali/Lesson02/grid_printer.py

```diff
@@ -0,0 +1,50 @@
+# Problem 1
+def createGrid1():
```

Comment:
make sure to use snake_case for var and function naming per python style guide -> `create_grid1`

Reply:
Thanks Natasha, I'll keep an eye on that.